cli/compile/compile.go

```diff
@@ -264,6 +269,28 @@ func runCompileCommand(cmd *cobra.Command, args []string) {
 	})
 	if compileError != nil && output.OutputFormat != "json" {
```

Comment:
All the added logic happens if `output.OutputFormat != "json"`, this means that no error is returned in case the output is json